src/index.js

```diff
@@ -1,3 +1,9 @@
+try {
```

Comment:
Interesting approach. I have been of the opinion that we should leave the responsibility of importing babel-polyfill to the application. 

This approach is convenient for the examples but could lead to confusion when multiple libs use ES6 etc. 

OK for now.